src/zope/schema/_bootstrapfields.py

```diff
@@ -564,6 +566,7 @@ def validate(self, value):
         return super(Password, self).validate(value)
 
 
+@implementer(IFromUnicode, IFromBytes)
 class Bool(Field):
```

Comment:
Hmm, does this need a `.. versionchanged::` bit?

Reply:
This isn't really here or there, but the fact that many of the fields (and interfaces) are in `_` prefixed files, but documented in `zope.schema` and `zope.schema.interface`  makes cross-referencing hard. Either you get a linkable cross-reference but use the internal name, or you document the preferred import but don't get a link. I wish I knew a way to have the cake and eat it too.